cpp/dolfinx/mesh/interval_mesh.cpp

```diff
@@ -73,7 +73,7 @@ mesh::Mesh build(MPI_Comm comm, std::size_t nx, std::array<double, 2> x,
 } // namespace
 
 //-----------------------------------------------------------------------------
-mesh::Mesh IntervalMesh::create(MPI_Comm comm, std::size_t n,
+mesh::Mesh interval_mesh::create(MPI_Comm comm, std::size_t n,
```

Comment:
No need for a namespace here. `mesh::create_interval` would be fine. Same for other cases.